core/server/api/app.js

```diff
@@ -11,7 +11,7 @@ var debug = require('debug')('ghost:api'),
     // API specific
     auth = require('../auth'),
     cors = require('../middleware/api/cors'),   // routes only?!
-    spamPrevention = require('../middleware/api/spam-prevention'), // routes only
```

Comment:
Plus if you haven't seen: we are also having an implementation of spam prevention here: https://github.com/TryGhost/Ghost/blob/master/core/server/apps/private-blogging/lib/middleware.js#L141

You could choose just creating an issue out of it, if the new logic can be re-used. I know it would create a dirty require, but we have to work on a solution for this anyway.